examples/00_basic/app.py

```diff
@@ -1,60 +1,121 @@
 import os
 from flask import Flask, jsonify, request, make_response
+from flask_sqlalchemy import SQLAlchemy
 from functools import wraps
+from passlib.hash import sha256_crypt as sc
+import uuid
+import secrets
 import jwt
 import datetime
 
 app = Flask(__name__)
 app.config['SECRET_KEY'] = os.environ.get('Secret')
+app.config['SQLALCHEMY _DATABASE_URI'] = 'something'
 
+db = SQLAlchemy(app)
+blacklist = []
+class User(db.Model):
+    ''' Setting up User'''
+    id = db.Column(db.Integer, primary_key=True)
+    # User vereification
+    email = db.Column(db.String(300), nullable=False, unique=True)
+    username = db.Column(db.String(60), nullable=False, unique=True)
+    password = db.Column(db.String(300), nullable=False)
```

Comment:
Storing a password in clear text is a bad idea. Better to add password functions to ensure hashing (and storing a password as a hash). In the User class, you can use the `@property` decorator to ensure a property is read-only.